packages/fast-components-styles-msft/src/carousel/index.ts

```diff
@@ -82,62 +91,62 @@ const styles: ComponentStyles<CarouselClassNameContract, DesignSystem> = (
             "& $carousel_flipperPrevious, & $carousel_flipperNext": {
                 color: "black",
                 fill: "black",
-                background: "white",
+                background: "rgba(255, 255, 255, 0.6)",
+                border: "1px solid rgba(206, 206, 206, 0.6)",
```

Comment:
same here, are none of these values based on the design system?

Reply:
See other response.